src/services/upload_repository.js

```diff
@@ -20,6 +20,7 @@ export default class UploadRepository {
 
   async uploadBundle(bundleId, storagePeriods) {
     const fee = await this.feesWrapper.feeForUpload(storagePeriods);
+    await this.feesWrapper.ensureEnoughFunds(fee);
```

Comment:
Yeah... You get `fee` from `feesWrapper` just to pass it again to `feesWrapper`, to check if it is enough.